src/components/course/CourseSidebarPrice.jsx

```diff
@@ -24,7 +26,7 @@ const CourseSidebarPrice = () => {
       <>
         {!enterpriseConfig.hideCourseOriginalPrice && (
           <div className="mb-2">
-            <del>${coursePrice.list} {currency}</del>
+            <del>${numberWithPrecision(coursePrice.list)} {currency}</del>
```

Comment:
nice

Reply:
yeah I decided that it's best to have the string conversion be done in the display code rather than in the calculation hook where we just output numbers now